Tasks/MavenAuthenticateV0/task.json

```diff
@@ -9,15 +9,22 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 0,
-        "Minor": 177,
-        "Patch": 1
+        "Minor": 184,
```

Comment:
Could you please bump task version to current sprint according to [steps](https://github.com/microsoft/azure-pipelines-tasks/blob/master/docs/taskversionbumping.md)?

Reply:
This is fixed in latest push.